templates/glossary-body-default.html

```diff
@@ -0,0 +1,145 @@
+<style>
+  ul {
+    -webkit-column-count: 3;
+    /* -moz-column-count: 3; */
+    column-count: 3;
+    float: left;
+    /* width: 33.33%; */
+  }
+</style>
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+
+<div id="glossary-body">
+  <div class='page-header'>
+    <h1 data-en-heading="Glossary" style="color:#002856">Vula Glossary</h1>
+  </div>
+
+  <div id="divTopTerms">
+    <div class="top-buttons">
+      <button type="button" id="viewAll" name="viewAll" class="btn btn-primary btn-xs pull-right">
+        View All Terms</button>
+      <button type="button" id="btn_new_term" name="btn_new_term" class="btn btn-success btn-xs pull-right"
+        data-action="new-term">
+        <i class="fa fa-plus" aria-hidden="true"></i> New term
+      </button>
+    </div>
+    <br />
+    {% foreach ($displayList as $x => $domain) { %}
+    {% if (count($domain['terms']) > 0) { %}
+    <section>
+      <div class="row">
+        <h4>Top {{ $domain['name'] }} Terms</h4>
+        {% foreach ($domain['terms'] as $x => $term) { %}
+        <!-- Existing term display code here -->
+        <div class="col-sm-4 col-md-4">
+          <div class="alert custom-alert" role="alert">
+            <div class="row d-flex justify-content-between align-items-center">
+              <div class="col-sm-8">
+                <strong>{{ $term['term'] }}</strong>
+              </div>
+              <div class="col-sm-4" id="action_buttons">
+                <button type="button" data-term_id="{{ $term['id'] }}" data-term_name="{{ $term['term'] }}"
```

Comment:
You don't need to include all the data element in each button - you can add it to the parent and then access from each button or add it to a hidden element which you can then just use the `$term['id']` to access.